plugins/content_distribution/providers/cortex/lib/CortexApiDistributionEngine.php

```diff
@@ -467,6 +467,16 @@ protected function uploadVideo(KalturaCortexApiDistributionJobProviderData $apiD
 		}
 		catch(Exception $e)
 		{
+			if ($needDel)
+			{
+				kFile::unlink($videoFilePath);
+			}
+			
+			if($isRemote)
```

Comment:
@amiras89 If an exception has been thrown before line 437, the var $isRemote still has not been set and it will print  a PHP notice, and also I would check if the file exists like that:
`if(isset($isRemote) && $isRemote && kFile::checkFileExists($tempVideoFilePath))`

Reply:
I suggest to simplify it to:
isset($tempVideoFilePath)

If exception is thrown before L438 - it won't be set == false.
If the file is not remote, it will be null --> which isset(null) is false